webservices/resources/candidates.py

```diff
@@ -194,14 +176,12 @@ def get(self, **kwargs):
 
         return data
 
-    def get_candidate(self, args, page_num, per_page, candidate_id, committee_id):
```

Comment:
It may end up changing somewhat, but we'll get there when we get there. :) Glad it's clear now.